frontend/src/pages/team/Home/components/InstanceStat.vue

```diff
@@ -43,28 +48,36 @@ export default {
                 return 'Running'
             case 'error':
                 return 'Error'
-            case 'not-running':
+            case 'stopped':
             default:
                 return 'Not Running'
             }
-        },
-        apiStates () {
-            switch (this.state) {
-            case 'running':
-                return ['starting', 'importing',
-                    'connected', 'info', 'success', 'pushing', 'pulling',
-                    'loading', 'installing', 'safe', 'protected', 'running', 'warning']
-            case 'error':
-                return ['error', 'crashed']
-            case 'not-running':
-            default:
-                return ['stopping', 'restarting', 'suspending', 'rollback', 'stopped', 'suspended', '']
-            }
+        }
+    },
+    methods: {
+        clicked () {
+            this.$emit('clicked', { type: this.type, state: this.state })
         }
     }
 }
 </script>
 
 <style scoped lang="scss">
+.ff-stat {
+    border: 1px solid transparent;
+    transition: ease-in-out .3s;
```

Comment:
```suggestion
    transition: ease-in-out .15s;
```

`0.3s` is a little too "soft", so changing it to 0.15